SQL succeeded but Query is now invalid ('task_editor', 283, 'accept'): 
UPDATE tasks 
SET do = 'refaktor alle ux in kleine module',
    priority = 0.0,
    level_id = 0,
    primary_activity_id = 2,
    secondary_activity_id = NULL,
    space_id = 1,
    ilk = 1
WHERE id=300
